doc/vim-go.txt

```diff
@@ -478,6 +503,20 @@ documentation in the |go-commands| section. Available <Plug> keys are:
 Calls `go run` for the current file
 
 
+The next three options are neovim only
```

Comment:
I think we should remove this line. Otherwise the word `three` needs to be updated every time we add a new feature or change something, and trust me that's not going to happen. I think best would be add something like: `(neovim only)` for each sentence